### ISSUE TITLE
Fails to work on Debian Sarge 9

### ISSUE THREAD

**Issue by stylon** [ringcentral/ringcentral-embeddable-electron-app#12]
Both the AppImage and the .deb package fail to run on a regular Debian Sarge installation. First it complains about chrome-sandbox not being SUID. After setting the SUID bit (on the .deb package) it crashes upon startup immediately with an 'Trace/breakpoint trap'.

**Comment by embbnux** [comment]
Thanks @stylon, I will try on Debian Sarge 9 and find how to fix it

**Comment by embbnux** [comment]
@stylon Which desktop environment are you using? I try to install debian based on Virtualbox, but need to select a desktop environment.

**Comment by stylon** [comment]
I'm using and tried only with GNOME3 on X.

**Comment by embbnux** [comment]
Got it

**Comment by embbnux** [comment]
@stylon Found a issue related to Electron https://github.com/electron/electron/issues/17972
You can run AppImage file with `--no-sandbox` to fix it:

```
chmod +x ringcentral-embeddable-voice-app-0.1.2-x86_64.AppImage
./ringcentral-embeddable-voice-app-0.1.2-x86_64.AppImage --no-sandbox
```

Still need some time for deb issue.

**Comment by embbnux** [comment]
Released 0.1.3, deb package should work in Debian. For AppImage file, user still need to run it with `--no-sandbox`
